app/src/main/java/com/example/kuba10/mypokemonplaces/PlaceDetailsFragment/PlaceDetailsFragment.java

```diff
@@ -1,180 +0,0 @@
-import com.example.kuba10.mypokemonplaces.R;
-import com.google.firebase.database.DatabaseReference;
-import com.google.firebase.database.Query;
-import com.squareup.picasso.Picasso;
-
-import java.util.ArrayList;
-
-public class PlaceDetailsFragment extends DialogFragment {
-
-
-    private TextView title, description;
-    public ImageView image, showLocation;
-    public ImageButton favouriteBtn;
-
-    private PokePlace userSelectedPlace;
-    private ArrayList<PokemonGo> pokemonGo_data_list;
-    private FirebaseListFragment parentFragment;
-    private Query query;
-
-    public static PlaceDetailsFragment newInstance(PokePlace place) {
-
-        PlaceDetailsFragment fragment = new PlaceDetailsFragment();
-        Bundle args = new Bundle();
-        args.putParcelable(Constants.DETAIL_POKEMON, place);
-        fragment.setArguments(args);
-
-        return fragment;
-    }
-
-    @Override
-    public void onStart() {
-        super.onStart();
-        setFullScreenStyle();
-    }
-
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        if (getArguments() != null) {
-
-            userSelectedPlace = getArguments().getParcelable(Constants.DETAIL_POKEMON);
-            parentFragment = (FirebaseListFragment) getFragmentManager().findFragmentByTag(Constants.LIST_FRAGMENT_TAG);
-            query = parentFragment.getQuery();
-        }
-    }
-
-    @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-                             Bundle savedInstanceState) {
-
-
-        View view = inflater.inflate(R.layout.fragment_place_details, container, false);
-
-        pokemonGo_data_list = parentFragment.sendPokemonListToAdapter();
-
-        title = (TextView) view.findViewById(R.id.details_title);
-        description = (TextView) view.findViewById(R.id.details_description);
-        image = (ImageView) view.findViewById(R.id.details_image);
-        showLocation = (ImageView) view.findViewById(R.id.details_show_on_map_button);
-        favouriteBtn = (ImageButton) view.findViewById(R.id.details_favourite_button);
-
-        image.setImageResource(R.drawable.ic_034_pikachu_1);
-        showLocation.setImageResource(R.drawable.ic_021_pointer);
-
-
-        title.setText(userSelectedPlace.getTitle());
-        description.setText(userSelectedPlace.getDesctription());
-
-        switch (userSelectedPlace.getFavourite()) {
-            case 0:
-                favouriteBtn.setImageResource(R.drawable.ic_032_star_empty);
-                break;
-            case 1:
-                favouriteBtn.setImageResource(R.drawable.ic_032_star);
-                break;
-
-        }
-
-        setPokemonImage();
-
-
-        favouriteBtn.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                switch (userSelectedPlace.getFavourite()) {
-
-                    case 0:
```

Comment:
To samo co wyżej. W ogóle widze, że w tej klasie powtarza się trochę metod z tych co były wcześniej. Może jakiś interfejs?